components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -53,6 +59,17 @@ export class ProjectListComponent implements OnInit {
     this.iamMajorVersion$ = store.select(iamMajorVersion);
     this.iamMinorVersion$ = store.select(iamMinorVersion);
 
+    this.applyRulesButtonText$ = this.projects.applyRulesStatus$.pipe(
+      map(({ state, percentageComplete }) => {
+        switch (state) {
+          case 'not_running':
+            return 'Update Projects';
+          case 'running':
```

Comment:
Could you grab the named constants for `not_running` and `running` from lib/authz/project_update_manager.go ?
Similar note for project.reducer.ts.